scaletest/terraform/action/coder_traffic.tf

```diff
@@ -0,0 +1,236 @@
+          command = concat([
+            "/opt/coder",
+            "--verbose",
+            "--url=${local.deployments.primary.url}",
+            "--token=${trimspace(data.local_file.api_key.content)}",
+            "exp",
+            "scaletest",
+            "workspace-traffic",
+            "--template=kubernetes-europe",
+            "--concurrency=0",
+            "--bytes-per-tick=${local.bytes_per_tick}",
+            "--tick-interval=${local.tick_interval}",
+            "--scaletest-prometheus-wait=30s",
+            "--job-timeout=${local.traffic_types[each.key].duration}",
+            "--workspace-proxy-url=${local.deployments.europe.url}",
+          ], local.traffic_types[each.key].flags)
+        }
+        restart_policy = "Never"
+      }
+    }
+  }
+  wait_for_completion = true
+
+  timeouts {
+    create = local.traffic_types[each.key].job_timeout
+  }
+
+  depends_on = [time_sleep.wait_baseline, time_sleep.wait_traffic[each.key]]
+}
+
+resource "kubernetes_job" "workspace_traffic_asia" {
+  provider = kubernetes.asia
+
+  for_each = local.traffic_types
+  metadata {
+    name      = "${var.name}-workspace-traffic-${each.key}"
+    namespace = kubernetes_namespace.coder_asia.metadata.0.name
+    labels = {
+      "app.kubernetes.io/name" = "${var.name}-workspace-traffic-${each.key}"
+    }
+  }
+  spec {
+    completions   = 1
+    backoff_limit = 0
+    template {
+      metadata {}
+      spec {
+        affinity {
+          node_affinity {
+            required_during_scheduling_ignored_during_execution {
+              node_selector_term {
+                match_expressions {
+                  key      = "cloud.google.com/gke-nodepool"
+                  operator = "In"
+                  values   = ["${google_container_node_pool.node_pool["asia_misc"].name}"]
+                }
+              }
+            }
+          }
+        }
+        container {
+          name  = "cli"
+          image = "${var.coder_image_repo}:${var.coder_image_tag}"
+          command = concat([
+            "/opt/coder",
+            "--verbose",
+            "--url=${local.deployments.primary.url}",
+            "--token=${trimspace(data.local_file.api_key.content)}",
+            "exp",
+            "scaletest",
+            "workspace-traffic",
+            "--template=kubernetes-asia",
+            "--concurrency=0",
+            "--bytes-per-tick=${local.bytes_per_tick}",
+            "--tick-interval=${local.tick_interval}",
+            "--scaletest-prometheus-wait=30s",
+            "--job-timeout=${local.traffic_types[each.key].duration}",
+            "--workspace-proxy-url=${local.deployments.asia.url}",
+          ], local.traffic_types[each.key].flags)
+        }
+        restart_policy = "Never"
+      }
+    }
+  }
+  wait_for_completion = true
+
+  timeouts {
+    create = local.traffic_types[each.key].job_timeout
+  }
+
+  depends_on = [time_sleep.wait_baseline, time_sleep.wait_traffic[each.key]]
```

Comment:
note for reviewers: these introduce a transitive dependency on the create-workspaces jobs.